certbot/main.py

```diff
@@ -623,7 +623,11 @@ def _handle_exception(exc_type, exc_value, trace, config):
             # Here we're passing a client or ACME error out to the client at the shell
             # Tell the user a bit about what happened, without overwhelming
             # them with a full traceback
-            err = traceback.format_exception_only(exc_type, exc_value)[0]
+            from dialog import DialogError
+            if issubclass(exc_type, DialogError):
```

Comment:
You should use `dialog.error` here. Dialog throws other error messages as well, which also have this problem.